checker/build.gradle

```diff
@@ -135,6 +137,21 @@ task updateJdk() {
         }
     }
 }
+''
```

Comment:
Out of curiosity: what does the `''` here do?

Reply:
Nothing!  I've removed them.  Thanks.